src/workerd/api/rtti.c++

```diff
@@ -148,14 +158,44 @@ CompatibilityFlags::Reader compileFlags(capnp::MessageBuilder &message, kj::Stri
   return kj::mv(reader);
 }
 
+CompatibilityFlags::Reader compileAllFlags(capnp::MessageBuilder &message) {
+  auto output = message.initRoot<CompatibilityFlags>();
+  auto schema = capnp::Schema::from<CompatibilityFlags>();
+  auto dynamicOutput = capnp::toDynamic(output);
+  for (auto field: schema.getFields()) {
+    bool isNode = false;
+
+    kj::StringPtr enableFlagName;
+
+    for (auto annotation: field.getProto().getAnnotations()) {
+      if (annotation.getId() == COMPAT_ENABLE_FLAG_ANNOTATION_ID) {
+        enableFlagName = annotation.getValue().getText();
+        // Exclude nodejs_compat, since the type generation scripts don't support node:* imports
+        // TODO: Figure out typing for node compat
+        isNode = enableFlagName == "nodejs_compat" ||
```

Comment:
`nodejs_als` _should_ influence the generated types, since it adds an additional module. However, this PR does not support automatic type generation for modules, and so `nodejs_als` has no affect. The reason we exclude the `node*` compat flags is primarily becasue they add things like `Buffer` to the global, which we don't want in the experimental entrypoint. This is very much not a perfect solution for node typings—we're working on a better one as we rethink what the future of node compat is with things like `nodejs_compat_v2`.